tests/unittests/client/test_runner.py

```diff
@@ -95,7 +95,7 @@ def new_runner(idle_timeout, n_workers=2, client=None):
     return runner
 
 
-def function_raise_on_2(lhs, rhs):
+def function_raise_on_2(lhs, sleep):
```

Comment:
What about the rhs variable below?

Reply:
What about it ?

Reply:
It is crying, looking for its identity, nowhere defined. :sob: